src/schemas/plugin-options.json

```diff
@@ -90,6 +90,34 @@
 			"type": "boolean",
 			"description": "Generate the sprite preview",
 			"link": "https://github.com/yoriiis/svg-chunk-webpack-plugin#generatespritespreview"
+		},
+		"injectSpritesInTemplates": {
+			"description": "specify the sprites map should be injected or not in a html webpack plugin",
+			"link": "https://github.com/yoriiis/svg-chunk-webpack-plugin#injectSpritesInTemplates",
+			"oneOf": [
+				{
+					"type": "boolean"
+				},
+				{
+					"type": "object",
+					"properties": {
+						"position": {
+							"link": "https://github.com/yoriiis/svg-chunk-webpack-plugin#injectSpritesInTemplates.position",
+							"description": "the position where to inject the sprites map: either at the beginning of the body or at the end",
+							"type": "string",
+							"enum": ["before", "after"]
+						},
+						"entries": {
+							"link": "https://github.com/yoriiis/svg-chunk-webpack-plugin#injectSpritesInTemplates.entries",
+							"description": "a list of entries we want to include the sprites on (in case of multiples entries generated by htmlWebpackPlugin",
+							"type": "array",
+							"items": {
+								"type": "string"
+							}
+						}
+					}
+				}
```

Comment:
Is it really necessary to make the injection position customizable? Before closing the body tag seems sufficient by default, doesn't it?

Reply:
> Is it really necessary to make the injection position customizable? Before closing the body tag seems sufficient by default, doesn't it?

I agree that it should be sufficient for most cases (in fact I don't even see what case may require to get the sprite-map at the bottom of body). For the record I started by using the htmlwebpackplugin api and pushing the tag into  bodytags then noticed it was adding to the end so I did the other part of code :)

Reply:
On HtmlWebpackPlugin, for styles or scripts, it's interesting to be able to choose the position (and even then, only for scripts). But for our purposes, an injection at the bottom of the DOM before closing the `</body>` tag seems sufficient. It should also simplify your code

Reply:
In fact sprites seems to work better when inserted at the first (eg: just after the opening `<body>` tag).

Reply:
Okay, so can we just keep the `before` injection as the default behavior and remove the customization?